Makefile

```diff
@@ -4,10 +4,15 @@ PWD?=$(subst /,\,${CURDIR})
 LDFLAGS=-s -w
 BLANK:=
 /:=\$(BLANK)
-LIFECYCLE_VERSION?=$(shell type VERSION)
 else
 /:=/
-LIFECYCLE_VERSION?=$(shell cat VERSION)
+endif
+
+ifeq ($(LIFECYCLE_VERSION),)
+LIFECYCLE_IMAGE_TAG?=$(shell git describe --always --dirty)
+LIFECYCLE_VERSION:=$(shell go run tools/version/main.go)
```

Comment:
Do we want to add `-dirty` to `LIFECYCLE_VERSION` as we do in `LIFECYCLE_IMAGE_TAG`?

Reply:
@ekcasey, we would love to know your opinion

Reply:
I honestly don't really have an opinion here. If pressed, I would say leave if off.